src/url-handling.js

```diff
@@ -229,6 +272,61 @@ export function stateFromLocation(location: Location): UrlState {
   };
 }
 
+export function updateStoreFromBrowserEvent({ dispatch, getState }: Store) {
```

Comment:
I had trouble understanding what this method was doing precisely, but I hope that `updateStoreFromBrowserEvent` is much clearer.